### PR TITLE
Editor styles and a few changes

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/89a38985e858052/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/89a38985e858052/output.txt

Total script time: 0.64 mins

Server is running on http://pr448.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/b0bb1e3f1616e1d/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/b0bb1e3f1616e1d/output.txt

Total script time: 0.50 mins

File: editors/default-editor.js [hunk 1]
```diff
@@ -163,17 +162,23 @@
...
-        if ( itemLabel === "In" ) {
-          itemLabel = "Start (seconds)";
-        } else if ( itemLabel === "Out" ) {
-          itemLabel = "End (seconds)";
+        // Add units, they exist in the manifest
+        if ( item === "start" ||  item === "end" ) {
+          currentItem.units = "seconds";
+        }
+        if ( currentItem.units ) {
+          unitLabel = document.createElement("span");
```
Comment (secretrobotron) [inline editors/default-editor.js:179]: Missed one!

File: editors/default-editor.js [hunk 2]
```diff
@@ -188,9 +193,23 @@
         // Remember first control added in editor so we can focus
         elemToFocus = elemToFocus || field;
 
+        //Add classes for style selecting
+        row.classList.add( rowClassPrefix + item );
+
+        //Add unit label if it exists
+        if( unitLabel ) {
+          col2.classList.add("butter-form-append");
```
Comment (secretrobotron) [inline editors/default-editor.js:201]: This one too.

File: editors/default-editor.js [hunk 3]
```diff
@@ -206,6 +225,9 @@
         }
       }
 
+      //Add the title info
+      document.getElementById("title").innerHTML = "<h3>"+e.data.manifest.about.name+"</h3>";
```
Comment (secretrobotron) [inline editors/default-editor.js:229]: And this one.

**Pull request merged**